app/controllers/distributions_controller.rb

```diff
@@ -30,7 +30,7 @@ def destroy
     if result.success?
       flash[:notice] = "Distribution #{params[:id]} has been reclaimed!"
     else
-      flash[:error] = "Could not destroy distribution #{params[:id]}. Please contact technical support."
+      flash[:error] = "Sorry, we weren't able to delete the distribution because that would reduce available inventory below zero."
```

Comment:
Are we sure that this is the only possible error that could happen? The text seems to indicate that.